README.md

```diff
@@ -4,6 +4,10 @@ This assistant lints text colors, border colors, fills, text sizes, and font fam
 Winter '21 version of the Salesforce Lightning Design System
 [design tokens](https://www.lightningdesignsystem.com/design-tokens/).
 
+## About Sketch Assistants
+
+First time using a Sketch Assitant? Read Sketch's help documentation [here](https://www.sketch.com/docs/assistants/). 
```

Comment:
Hey @ryukelson1 - just a heads up that for accessibility we advise people to put more meaningful text into links, so that is meaningful even out of context. For example, instead of "here" or "read more" you'll have the link text be something like "Read Sketch's help documentation". https://www.washington.edu/accessibility/links/

Reply:
Also, "Assistant" is spelled wrong on line 9.